src/namicorn.ts

```diff
@@ -24,7 +25,8 @@ export default class Namicorn {
   readonly ens?: Ens;
   /** @internal */
   readonly zns?: Zns;
-  /** @internal */
+  /** @ignore */
+  readonly cns?: Cns;
   readonly api?: Udapi;
```

Comment:
There was an `@internal` tag for `api` property.  Don't we still need it?

Reply:
Yes, we are now using `@internal` but never `@ignore`.